README.md

```diff
@@ -56,6 +56,54 @@ If you create a feature request, use the dedicated **Feature request** issue tem
 
 If you have a bug to report, reproduce it in an online code editor. For example, use [CodeSandbox](https://codesandbox.io/). Attach the link to the reproduced bug to your issue. Log the bug using the **Bug report** template.
 
+## Releasing
+
```

Comment:
Could you add an introductory sentence, like `Follow these steps to create a changelog for the new release.` or sth like that?